### PR TITLE
[DROOLS-936] fix duplicated classes

### PR DESCRIPTION
**Pull request by psiroky**

### CODE REVIEW

File: drools-wb-jcr2vfs-migration/drools-wb-jcr2vfs-export/pom.xml [hunk 1]
```diff
@@ -270,6 +299,39 @@
...
+                <rules>
+                  <banDuplicateClasses>
+                    <dependencies>
+                      <dependency>
+                        <groupId>org.drools</groupId>
+                        <artifactId>guvnor-jcr2vfs-migration-webapp-drools-as-jar</artifactId>
+                        <ignoreClasses>
+                          <!-- Legacy dependency, will never get fixed. -->
+                          <ignoreClass>org.drools.guvnor.client.resources.WizardResources$WizardStyle</ignoreClass>
+                            <ignoreClass>org.drools.guvnor.client.resources.WizardResources</ignoreClass>
```
Comment (manstis) [inline drools-wb-jcr2vfs-migration/drools-wb-jcr2vfs-export/pom.xml]: If I was picky I'd moan about the formatting (L#323 is indented one too many)

File: drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-backend/pom.xml [hunk 2]
```diff
@@ -106,10 +112,10 @@
       <artifactId>guvnor-project-api</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>javax.enterprise</groupId>
-      <artifactId>cdi-api</artifactId>
-    </dependency>
+    <!--<dependency>-->
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-backend/pom.xml]: Please remove commented code.

File: drools-wb-jcr2vfs-migration/drools-wb-jcr2vfs-export/pom.xml (hunk 1 above)
Comment (psiroky) [inline drools-wb-jcr2vfs-migration/drools-wb-jcr2vfs-export/pom.xml]: @manstis please be picky :) Thanks for spotting that, will fix it.

File: drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-backend/pom.xml (hunk 2 above)
Comment (psiroky) [inline drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-backend/pom.xml]: Thanks, will remove it.

**Pull request merged**